### PR TITLE
Change custom grunt protractor task to third party one

### PR DESCRIPTION
**Pull request by RomanKonovalov**
Change custom grunt protractor task to third party one

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
                 src: 'src/main/webapp/js/min/script.js',
                 dest: 'src/main/webapp/js/min/script.clean.js'
             }
+        },
+        protractor : {
+            options : {
+                keepAlive : true,
```
Comment (UladzimirKapylou) [inline Gruntfile.js]: Will this keep protractor alive always or only in case of fail?

File: Gruntfile.js [hunk 2]
```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
...
+                noColor : false,
```
Comment (UladzimirKapylou) [inline Gruntfile.js:172]: What is intent of using `noColor` option?

File: Gruntfile.js (hunk 1 above)
Comment (RomanKonovalov) [inline Gruntfile.js]: If false, the grunt process stops when the test fails. 
For now (jennkins is configured and e2e tests are passed) I think we should switch it to false

Comment (UladzimirKapylou) [inline Gruntfile.js]: Ok, but you set value to `true`. What is the behaviour in this case?

File: Gruntfile.js (hunk 2 above)
Comment (RomanKonovalov) [inline Gruntfile.js:172]: If true, protractor will not use colors in its output.
I guess 'true' is more suitable

Comment (UladzimirKapylou) [inline Gruntfile.js:172]: Why do you think so?

File: Gruntfile.js (hunk 1 above)
Comment (RomanKonovalov) [inline Gruntfile.js]: When 'keepAlive : true' maven task (grunt protractor) never fails irrespective of the protractor results

File: Gruntfile.js (hunk 2 above)
Comment (RomanKonovalov) [inline Gruntfile.js:172]: In pom file we use <arguments>--no-color</arguments> in 'frontend-maven-plugin' configuration

File: Gruntfile.js (hunk 1 above)
Comment (UladzimirKapylou) [inline Gruntfile.js]: Cool! That's great!

File: Gruntfile.js (hunk 2 above)
Comment (UladzimirKapylou) [inline Gruntfile.js:172]: Ok, but why colors are so bad? They should do logs more readable in console.

Comment (RomanKonovalov) [inline Gruntfile.js:172]: To switch on colors we should remove --no-color flags in pom file and grunt file. So maybe it's work for another PR?

Comment (UladzimirKapylou) [inline Gruntfile.js:172]: I think so. But I thought you know the reason of such configuration.

**Comment by RomanKonovalov** [comment]
"retest this please"

**Comment by SviatlanaHlukhava** [comment]
:+1:

**Comment by UladzimirKapylou** [comment]
:+1:

**Pull request merged**